dodo.py

```diff
@@ -1506,6 +1650,13 @@ def _ensure_resolutions(app_name):
 
         return all_up_to_date
 
+    def fetch_pyodide_repodata():
+        url = "https://cdn.jsdelivr.net/pyodide/v0.23.0/full/repodata.json"
```

Comment:
Hmm does that means we also need to know the Pyodide version here? Which is already handled in https://github.com/jupyterlite/pyodide-kernel.

Looks like one more piece of configuration that will need to be updated next time we update the pyodide kernel.